backend/src/main/java/dev/tripdraw/domain/member/Member.java

```diff
@@ -46,6 +57,17 @@ public static Member of(String oauthId, OauthType oauthType) {
         return new Member(null, null, oauthId, oauthType);
     }
 
+    public void delete() {
+        validateNotDeleted();
+        this.isDeleted = TRUE;
+    }
+
+    private void validateNotDeleted() {
+        if (isDeleted == TRUE) {
+            throw new MemberException(MEMBER_NOT_FOUND);
+        }
```

Comment:
다음과 같이 간소화 시킬 수 있을 것 같아요! 가독성 측면에서도 더 좋구요
```suggestion
        if (isDeleted) {
            throw new MemberException(MEMBER_NOT_FOUND);
        }
```

Reply:
나 바본가